relay-server/src/services/store.rs

```diff
@@ -184,6 +186,15 @@ impl StoreService {
     ) -> Result<(), StoreError> {
         let retention = envelope.retention();
         let event_id = envelope.event_id();
+
+        let feedback_ingest_topic_rollout_rate = self
+            .global_config
+            .current()
+            .options
+            .feedback_ingest_topic_rollout_rate;
+        let use_feedback_topic =
+            is_rolled_out(scoping.organization_id, feedback_ingest_topic_rollout_rate);
```

Comment:
This will cause the rollout load volume to be unbalanced.  Large orgs will send significantly more messages than small orgs.  This might be okay.  Be aware going from 10% -> 20% does not necessarily indicate a 2x increase in volume.

Reply:
10 -> 20 will equal a 2x increase in volume from any _given org_, right? But not overall.